solidity/package.json

```
@@ -3,8 +3,9 @@
 	"type": "module",
 	"description": "Augur-Constant-Product-Market-Contracts",
 	"scripts": {
+		"contracts": "npm ci --ignore-scripts && npm run compile && cd .. && npm run vendor",
 		"compile": "tsc && node ./js/compile.js",
-		"test": "tsc && node ./js/tests/test.js"
+		"test": "tsc --noEmit && node --import tsx $npm_config_testargs ./ts/tests/test.ts"
```

I don't think you need (or want) to compile here.  IIUC, `tsx` does in-memory compilation so you can run TS directly in NodeJS, so compiling first is just wasting time.
```suggestion
		"test": "npx tsx --test $npm_config_testargs"
```